INSTALLATION.md

```diff
@@ -1,76 +1,76 @@
+* If you have an Intel CPUs (i5, i7, ...), download ADM installer [here](https://github.com/massalabs/thyra/releases/download/v0.1.12/thyra-installer_darwin_amd64.tar.gz)
+2. Once downloaded, double clik on the file
+> **_NOTE:_** You may need to authorise the download of this file as your Mac won't recognise it. You can authorise the installer to run by right clicking on the installer and click on `Open` from Finder. A dialog box will open asking you if you are sure, press the `Open` button and the installer should start.
+3. A terminal should open automatically where you can follow the steps of the installation. Just watch it working, and at some point you'll need to add your password.
+* Why do you need to enter your password? Simply because Thyra is an application and to add this application to your Mac you need to validate that you wish to install this app; like you do for app from the app store. 
+4. Once the installation is finished, you will see a new icon on the top nav bar (see screenshot below). You can now close your terminal (if not done already) and start browsing the blockchain and all our applications available.
+<img width="177" alt="image" src="https://user-images.githubusercontent.com/109611779/223798813-92a16141-19f6-415b-b9d8-2554eb814edc.png">
 
-Simply use copy/paste the cmd line below in your terminal. The installation process will start and a success message will be displayed once done.
 
+> **_NOTE:_** If you quit Thyra, the icon tray will disappear. Normal! If you want to add it back, open your finder and look for "thyra" in all your mac. You should see the following file "XXXX"; click on it and the app will be back on your top nav bar. 
 
-```sh
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/massalabs/thyra/main/scripts/macos_install.sh)"
-```
+> **_TROUBLESHOOTING:_** Do not hesitate to report any problem or request help by [opening an issue](https://github.com/massalabs/thyra/issues/new).
 
-> **_NOTE:_** To complete the installation, you have to restart your browser.
 
-Congratulation, your version of Thyra is now installed on your system and can be run by executing `thyra-server` in your terminal.
-You can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](https://flappy.massa) can help.
-If you want to get down to business, you can start your [Massalian journey](https://my.massa/thyra/wallet) right away!
+🎉 Congratulations!! Your gate to Massa blockchain is now installed and you can start your massalian journey.
+You can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](http://flappy.massa) can help.
+If you want to get down to business, you can start your [Massalian journey](http://my.massa/thyra/wallet) right away!
```

Comment:
Why change it back to `http` ? If we want to push users to use best practices, we should put `https` on every possible links.  
```suggestion
You can now browse the **websites on-chain** seamlessly. If you need to take the pressure off, maybe a little [game](https://flappy.massa) can help.
If you want to get down to business, you can start your [Massalian journey](https://my.massa/thyra/wallet) right away!
```